server/services/visitService.ts

```diff
@@ -38,11 +38,15 @@ export default class VisitService {
     const token = await this.hmppsAuthClient.getSystemClientToken()
     const orchestrationApiClient = this.orchestrationApiClientFactory(token)
 
-    const { contact, phoneNumber } = bookingJourney.mainContact
-    const visitContact = {
-      name: typeof contact === 'string' ? contact : `${contact.firstName} ${contact.lastName}`,
-      ...(phoneNumber && { telephone: phoneNumber }),
-    }
+    const contact = bookingJourney.mainContact?.contact
```

Comment:
This method can now be called early in the journey as well as at the end so basically just tweaking it to handle some of the session data not being present.